src/Workspaces/VisualBasic/Portable/Formatting/Rules/AdjustSpaceFormattingRule.vb

```diff
@@ -6,9 +6,13 @@ Imports Microsoft.CodeAnalysis.Options
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
+#If Not CODE_STYLE Then
     <ExportFormattingRule(AdjustSpaceFormattingRule.Name, LanguageNames.VisualBasic), [Shared]>
     <ExtensionOrder(After:=ElasticTriviaFormattingRule.Name)>
     Friend Class AdjustSpaceFormattingRule
```

Comment:
❓ Are we really gaining anything here by using MEF? Several changes would disappear if I could remove this.

Reply:
➡️ I couldn't find any use of `ExportFormattingRuleAttribute` outside Roslyn so I removed this to simplify code.